src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -217,6 +220,7 @@ def _create_database_account(client,
                              assign_identity=None,
                              default_identity=None,
                              backup_policy_type=None,
+                             continuous_tier=None,
```

Comment:
```suggestion
                             continuous_tier='Continuous30Days',
```
From the code logic below, you want to set a default value for `continuous_tier`, right?

Reply:
we want to avoid setting the default value here since this "tier" should only be set when backup_policy_type is "continuous"